cpg-core/src/main/kotlin/de/fraunhofer/aisec/cpg/TranslationManager.kt

```diff
@@ -252,9 +287,40 @@ private constructor(
             )
         }
 
+        // Distribute all files by there root path prefix, parse them in individual component
+        // named
+        // like their rootPath local name
+        ctx.config.includePaths.forEach { includePath ->
+            val filesInPath =
+                importedSources.filter {
+                    it.path.removePrefix(includePath.toString()) != it.path.toString()
+                }
+            if (filesInPath.isNotEmpty()) {
+                val component = Component()
+                component.name = Name(includePath.name)
+                result.addComponent(component)
+
+                usedFrontends.addAll(
+                    if (useParallelFrontends) {
+                        parseParallel(component, result, ctx, filesInPath)
+                    } else {
+                        parseSequentially(component, result, ctx, filesInPath)
+                    }
+                )
+            }
+        }
+
         return usedFrontends
     }
 
+    private fun extractConfiguredSources(path: Path): MutableList<File> {
+        val rootFile = path.toFile()
+        return if (rootFile.exists())
```

Comment:
Please refactor this code. Add some `{` as a start.

Reply:
Did so too